test/fixtures/computed_values/outputs.tf

```
@@ -1,4 +1,4 @@
-/**
+/**pubsub-project-local
```

This appears to have been mistakenly added.

```suggestion
/**
```